articles/article-84-turing-award.md

```diff
@@ -0,0 +1,28 @@
+<!-- BEGIN TITLE -->
+# Why no Nobel prize for computer scienctists?
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Some people say it is because that Alfred Nobel’s fiancée had an affair with a mathematician, others say because he was more interested in Physics, Chemistry, Medicine, Literature, and Peace than in Mathematics, and computer science is a part of the Mathematics field. 
```

Comment:
Some say it was because Alfred Nobel's fiancé had an affair with a mathematician, which can be considered the foundation of computer science, while others believe Nobel was just more interested in physics, chemistry, medicine, literature and peace than in mathematics. 
- Alfred Nobel also lived in the 1800s, so the idea he did not like computer science is absurd. The earliest known computer was known as the Analytical machine, which was a glorified calculator. The first programmable computers did not become relevant until the 1900s. That is why I rephrased it in such a way that the focus was on mathematics.